pkg/scalers/gcp_pubsub_scaler.go

```diff
@@ -36,8 +36,9 @@ type pubsubScaler struct {
 }
 
 type pubsubMetadata struct {
-	mode  string
-	value int64
+	mode                  string
+	value                 int64
+	activationTargetValue int64
```

Comment:
please, use `activationValue`. The convention is already existing key with the prefix `activation`